src/webui/webapplication.cpp

```diff
@@ -872,14 +868,8 @@ void WebApplication::action_command_addCategory()
     CHECK_URI(0);
     CHECK_PARAMETERS("category");
 
-    QString category = request().posts["category"].trimmed();
-
-    if (!BitTorrent::Session::isValidCategoryName(category) && !category.isEmpty()) {
-        status(400, tr("Incorrect category name"));
-        return;
-    }
-
-    BitTorrent::Session::instance()->addCategory(category);
+    BitTorrent::Session::instance()->createCategory(
+                request().posts["category"].trimmed());
```

Comment:
here too

Reply:
Here the same case... I didn't ponder much into this logic. I just left the same behavior as before.